app-games/src/main/java/com/aptoide/android/aptoidegames/home/GamesScreen.kt

```diff
@@ -34,7 +34,7 @@ fun gamesScreen() = ScreenData(
 private fun GamesScreenContent(
   navigate: (String) -> Unit
 ) {
-  val showHomeTabRow = rememberHomeTabRowState()
+  val (showHomeTabRow, tabs) = rememberHomeTabRowState()
   var selectedTab by rememberSaveable(key = defaultHomeTabs.size.toString()) { mutableIntStateOf(0) }
```

Comment:
shouldn't it grab from the tabs instead of the default ones? Just asking this because the size might be different, which might lead to index out of bounds